src/loader.ps1

```diff
@@ -208,7 +210,17 @@ function getSettings() {
     if (!$isSettingsFromFile) {
         $newSettings = [ExomodLoaderSettingsFile]::new()
         $newSettings.SettingsVersion = 1;
-        $newSettings.PathToGameStoriesFolder = Read-Host -Prompt "Please enter the path of your game's unmodified Stories directory. Example: $(${ENV:ProgramFiles(x86)})\Steam\steamapps\common\Exocolonist\Exocolonist_Data\StreamingAssets\Stories"
+
+        # Prompt the user for the path to their game folder. Use the default path if none is provided.
+        $userGameDirectory = Read-Host -Prompt "Please enter the path of your game directory. Press enter to use the default value: $($defaultPathToGameFolder)"
```

Comment:
"Please enter [...] Press enter..."
Can we vary this a bit for readability? Examples: "Please provide..." or "Press the ENTER key..."

Reply:
How about
```suggestion
        $userGameDirectory = Read-Host -Prompt "Please provide the path of your game directory. Leave blank to use the default value: $($defaultPathToGameFolder)"
```